blip-project/src/pages/Signup/signup.jsx

```
@@ -0,0 +1,445 @@
+import Input from "../../components/SignUp, Login/input.jsx";
+import backgroundImg from "../../svg/backgroundImg (5).svg";
+import { color } from "../../style/color.jsx";
+import { PassWord } from "../../components/SignUp, Login/password.jsx";
+import { Email } from "../../components/SignUp, Login/email.jsx";
+import Id from "../../components/SignUp, Login/id.jsx";
+import { PasswordCheck } from "../../components/SignUp, Login/passwordCheck.jsx";
+import modalX from "../../svg/modalX.svg";
+import axios from "axios";
+import apiSignUp from "../../apis/apiSignUp.jsx";
+import sends from "../../svg/send.svg";
+import Number from "../../components/SignUp, Login/number.jsx";
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)